src/bitcoin-cli.cpp

```diff
@@ -338,8 +338,8 @@ static UniValue CallRPC(BaseRequestHandler *rh, const std::string& strMethod, co
 
     // check if we should use a special wallet endpoint
     std::string endpoint = "/";
-    std::string walletName = gArgs.GetArg("-rpcwallet", "");
-    if (!walletName.empty()) {
+    if (!gArgs.GetArgs("-rpcwallet").empty()) {
+        std::string walletName = gArgs.GetArg("-rpcwallet", "");
```

Comment:
Why this change? To reduce `walletName` scope?

Reply:
> Why this change? To reduce walletName scope?

Yes and to avoid declaring a variable that will never be used.